src/parameter.rs

```diff
@@ -13,6 +13,43 @@ pub(crate) struct Parameter<'src> {
   pub(crate) name: Name<'src>,
 }
 
+impl<'src> Parameter<'src> {
+  pub(crate) fn format_eval<'this, 'eval, 'run>(
+    &'this self,
+    evaluator: &'eval std::cell::RefCell<Evaluator<'src, 'run>>,
+  ) -> FmtEval<'this, 'eval, 'src, 'run> {
+    FmtEval(self, evaluator)
+  }
+}
+
+pub struct FmtEval<'param, 'eval, 'src, 'run>(
+  &'param Parameter<'src>,
+  &'eval std::cell::RefCell<Evaluator<'src, 'run>>,
+);
+
+impl<'this, 'eval, 'src, 'run> ColorDisplay for FmtEval<'this, 'eval, 'src, 'run> {
+  fn fmt(&self, f: &mut Formatter, color: Color) -> fmt::Result {
+    if self.0.export {
+      write!(f, "$")?;
+    }
+    if let Some(prefix) = self.0.kind.prefix() {
+      write!(f, "{}", color.annotation().paint(prefix))?;
+    }
+    write!(f, "{}", color.parameter().paint(self.0.name.lexeme()))?;
+    if let Some(ref default) = self.0.default {
+      write!(
+        f,
+        "={}",
+        color.string().paint(&format!(
```

Comment:
This is mainly because regular strings are green, including their quotes, and I didn't look too far into the coloring module.

Reply:
Unfortunately, I selected the wrong range here and GitHub won't let me correct it, refering to L43-L47.